storage/include/s3/metrics.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include <Metrics.hpp>
+
+namespace concord::storage::s3 {
+
+struct Metrics {
+  Metrics()
+      : metrics_component_{concordMetrics::Component("s3", std::make_shared<concordMetrics::Aggregator>())},
+        numKeysTransferred{metrics_component_.RegisterCounter("keys_transferred")},
+        bytesTransferred{
+            metrics_component_.RegisterCounter("bytes_transferred"),
+        },
+        lastSavedBlockId{
+            metrics_component_.RegisterStatus("last_saved_block_id", "0"),
+        }
+
+  {
+    metrics_component_.Register();
+  }
+
+  void updateLastSavedBlockId(const std::string& key) {
```

Comment:
The keys are naturally stored as `Keys` which is an alias to `Sliver`.
Can we use it instead of `std::string` and using Key::toString()?
AFAIK, `Sliver` also can also return a `string_view`.

Reply:
The reason I put std::string as input parameter is that we strictly threat the key as a string in this case and if for some reason it stops being a string in the future - the method will make no sense.

But it will be more consistent with the rest of the code if I get a Sliver. I'll change it.